packages/tools/u-boot/package.mk

```diff
@@ -35,8 +35,7 @@ esac
 
 post_patch() {
   if [ -n "$UBOOT_SYSTEM" ]; then
-    # Enable LZ4, LZO and LZMA decompression support
-    echo -e "CONFIG_LZ4=y\nCONFIG_LZO=y\nCONFIG_LZMA=y" >> "$PKG_BUILD/configs/$($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM config)"
+    find_file_path bootloader/config && cat $FOUND_PATH >> "$PKG_BUILD/configs/$($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM config)"
```

Comment:
Append `|| true`, or convert this one line to a 3-line `if`/`fi`, otherwise `post_patch()` will exit with a value of 1 when `find_file_path` does *not* find a matching file.

Reply:
good catch, fixed.